employee_mgmt/app/models/employee.rb

```diff
@@ -1,30 +1,60 @@
 class Employee < ActiveRecord::Base
 
-  def update_phoneno val
+ validates :first_name, presence:true
+  validates :last_name, presence: true
+  validates :phone,allow_blank: true, format: { with: /9[0-9]{9}/,message:"Must have exactly 10 digits number starting with 9"}
+ validates :salary, numericality: {greater_than: 0}
+ validates :email,uniqueness: true,allow_blank: true, format: { with: /[a-zA-Z]+@[a-z0-9A-Z]+.[.a-zA-Z]+/,message:"%{value}Invalid Email ID"}
+
+ after_validation(on: :create) do
+   self.first_name=self.first_name.capitalize
+   if(self.middle_name)
```

Comment:
we don't use parenthesis`()`  for simple conditions like this simply write `if self.middle_name`